diff --git a/wrappers/Python/setup.py b/wrappers/Python/setup.py
index 8a3e9234..aa9b9779 100644
--- a/wrappers/Python/setup.py
+++ b/wrappers/Python/setup.py
@@ -46,17 +46,6 @@ def recursive_collect_includes():

 if __name__=='__main__':

-    # Trying to change the standard library for C++
-    import platform
-    try:
-        macVersion = platform.mac_ver()[0].split('.')
-        if int(macVersion[0]) >= 10 and int(macVersion[1]) > 8:
-            os.environ["CC"] = "gcc"
-            os.environ["CXX"] = "g++"
-            print('switching compiler to g++ for OSX')
-    except:
-        pass
-
     # ******************************
     #       CMAKE OPTIONS
     # ******************************
@@ -148,8 +137,8 @@ if __name__=='__main__':
         else:
             raise ValueError('cmake_compiler [' + cmake_compiler + '] is invalid')

-        if 'darwin' in sys.platform:
-            cmake_config_args += ['-DCOOLPROP_OSX_105_COMPATIBILITY=ON']
+        # if 'darwin' in sys.platform:
+        #     cmake_config_args += ['-DCOOLPROP_OSX_105_COMPATIBILITY=ON']
         if 'linux' in sys.platform:
             cmake_config_args += ['-DCOOLPROP_FPIC=ON']
         #if sys.platform.startswith('win'):
